pkg/webui/console/views/app/index.js

```diff
@@ -136,6 +153,15 @@ class ConsoleApp extends React.PureComponent {
 
     return (
       <React.Fragment>
+        {status.isLoginRequired && (
+          <Modal
+            approval={false}
+            buttonMessage={m.buttonMessage}
+            message={m.modalMessage}
+            title={m.modalTitle}
+            onComplete={reload}
+          />
+        )}
```

Comment:
See my other comment above to source this out into a separate container.

```suggestion
        <LogBackInModal />
```